builtin/credential/github/path_login.go

```diff
@@ -177,6 +171,26 @@ func (b *backend) verifyCredentials(ctx context.Context, req *logical.Request, t
 		client.BaseURL = parsedURL
 	}
 
+	if config.OrganizationID == 0 {
+		// Previously we did not verify using the Org ID. So if the Org ID is
+		// not set, we will trust-on-first-use and set it now.
```

Comment:
Would be good to add a log message here to indicate that we're setting the organization ID on a trust-on-first use basis (for upgrade purposes).